.github/workflows/prepare-release.yml

```diff
@@ -1,8 +1,11 @@
-name: Update version on new release branch
+name: Update version and presets page on new release branch
 
 on:
   create:
 
+env:
+  SCARB_VERSION: 2.5.3
```

Comment:
what's our strategy to maintain this up to date?

Reply:
My idea is to continue using the same we've been using: update this version at the same time we update the Scarb.toml. We had to do this already for the `lint_and_test` action.

Reply:
i see. does it make sense to automate it in some way? what's preventing us from mistakenly not updating it?

Reply:
Done!